TestsFoundation/TestsFoundation/Extensions/XCUIElementExtensions.swift

```diff
@@ -42,6 +42,7 @@ public extension XCUIElement {
         case swipeDown
         case swipeRight
         case swipeLeft
+        case selfSwipeLeft
```

Comment:
Maybe adding an associated value to the `swipeLeft` case would be more elegant. This associated value could have a type name `Target` with 2 values: `onApp`, `onElement`, you could make it as an inner enum within the `ElementAction` enum.